app/services/submit_application.rb

```diff
@@ -10,11 +10,7 @@ def initialize(application_form)
 
   def call
     ActiveRecord::Base.transaction do
-      if application_form.apply_again? && enough_references_have_been_provided?
-        application_form.update!(submitted_at: Time.zone.now, edit_by: Time.zone.now)
-      else
-        application_form.update!(submitted_at: Time.zone.now, edit_by: edit_by_time)
-      end
+      application_form.update!(submitted_at: Time.zone.now, edit_by: edit_by_time)
```

Comment:
My only concern here is that `edit_by` gets out of sync with the 'state' of the application. Like you say it doesn't fail any specs but could we make assumptions in future based on `edit_by` that are broken for apply again applications?

Reply:
Yeah, I'll look at this separately when I'll look at https://ukgovernmentdfe.slack.com/archives/CN1MCQCHZ/p1590568483055500. It might have gthe same solution.